Lib/glyphsLib/builder/instances.py

```diff
@@ -274,44 +274,51 @@ def __init__(self, descriptor):
                 self.customParameters.append(GSCustomParameter(name, value))
 
 
-def _set_class_from_instance(ufo, designspace, instance, axis_def):
+def _set_class_from_instance(ufo, designspace, instance, axis_tag):
     # FIXME: (jany) copy-pasted from above, factor into method?
-    design_loc = None
+    assert axis_tag in ("wght", "wdth")
+
+    factory = AxisDefinitionFactory()
```

Comment:
This factory is meant to be used by calling `get` exactly once per axis in the same order as in the designspace/font, so that the Glyphs properties `weightValue`, `widthValue`, `customValue`, etc are "aligned" with the axes. It may not matter here but still, I would rather have the factory stay hidden in axes.py and provide a method like `get_designspace_axis_defs` for use outside of axes.py

What do you think?

Reply:
sure I can do that